Application Lifecycle Management/instances-api.md

```diff
@@ -2956,3 +2957,44 @@ ElasticBox-Release: 4.0
   "method":"cancel_import"
 }
 ```
+
+## PUT /services/instances/{instance_id}/abort_operation
+
+Abort a processing operation executing scripts on an instance when you give its ID.
+
+### URL
+
+#### Structure
+
+```
+[PUT] /services/instances/{instance_id}/abort_operation
+```
+
+#### Example
+
+```
+[PUT] https://cam.ctl.io/services/instances/i-ndt46z/abort_operation
+```
+
+### Request
+
+#### Headers
+```
+Content-Type: application/json
+Authorization: Bearer your_json_web_token
+ElasticBox-Release: 4.0
+```
+
+#### URI Parameters
+| Parameter | Type | Description | Req.|
+|-----------|------|------------|----|
+| instance_id | string | Instance id | Yes |
+### Response
```

Comment:
Please, leave 1 blank line above and below titles

Reply:
Done